### DISCUSSION TITLE
automatic logout setting

### DISCUSSION THREAD

**Discussion by fireball74** [Leantime/leantime#514] [Q&A]
When using Firefox I find that users are logged out after a shorter period of inactivity. The setting $sessionExpiration = 28800 does not seem to be respected at all. Is this known or better file a bug report?

**Comment by marcelfolaron** [comment]
Can you file a bug for that. Maybe someone can dive deeper. The variable is exactly the one that should be updated.

**Reply by fireball74** [reply]
#568 filed.
